h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -682,7 +682,7 @@ public static void main( String[] args ) {
             }
           } else if((c0 instanceof C8Chunk) && (c1 instanceof C8Chunk)) {
             long d0 = c0.at8(rows), d1 = c1.at8(rows);
-            if (d0 != d1) {
+            if (!(Double.isNaN(d0) && Double.isNaN(d1)) && !(Math.abs(d0-d1)<=Math.abs(d0+d1)*_epsilon) ) {
```

Comment:
wait, d0 and d1 are longs, how can they possibly be nans? Also, since they are longs (timestamps, you say), there's no reason to compare up to a scale. Can set some tolerance variable in the tests, or pass it as an argument. See, a long millis is a very big number; right now is 1487113772078, so your delta will be about 8 hours, ouch.